TCAT/Controllers/RouteDetailViewController.swift

```diff
@@ -95,7 +95,7 @@ class RouteDetailViewController: UIViewController, GMSMapViewDelegate, CLLocatio
                 
                 var waypoints: [Waypoint] = []
                 for point in path {
-                    var type: WaypointType = .none
+                    var type: WaypointType = .walking
                     if direction == self.directions.first && point == path.first! {
```

Comment:
Don't assume first exists. do an if let please

Reply:
If the for loop fires, there must be at least one element. That's how you get EXCITED in Swift properly